_data/sites.json

```
@@ -9481,6 +9481,16 @@
         ]
     },
 
+    {
+        "name": "MSI",
+        "url": "https://www.account.msi.com/",
```

```suggestion
        "url": "https://account.msi.com/",
```
Previous link was broken, this seems to do the trick